package.json

```diff
@@ -23,7 +23,9 @@
     "core-js": "~2.5.5",
     "css-loader": "~2.1.1",
     "es6-promise": "~4.0.5",
+    "eslint": "^6.8.0",
     "file-loader": "~1.1.11",
+    "glob": "^7.1.6",
```

Comment:
Please, use `fast-glob` or even `globby`, cause it faster than `glob`.

Both have similar syntax.